receipts/receipts.go

```diff
@@ -103,7 +103,8 @@ func Request(r xml.TokenReader) xml.TokenReader {
 			switch {
 			case t.Name.Local == "receipt" && t.Name.Space == NS:
 				noWrite = true
-			case t.Name.Local == "message" && (t.Name.Space == ns.Client || t.Name.Space == ns.Server):
+			case t.Name.Local == "message" && (t.
```

Comment:
Please remove the changes to this file; the commit should only touch the carbons package. Thanks!